modules/viewing-metrics-for-the-single-model-serving-platform.adoc

```diff
@@ -14,7 +14,7 @@ ifdef::self-managed[]
 endif::[]
 ifdef::cloud-service[]
 * You have access to the OpenShift cluster as a developer or as a user with view permissions for the project that you are viewing metrics for.
-* You are familiar with querying metrics in user-defined projects. See link:https://docs.openshift.com/dedicated/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS).
+* You are familiar with querying metrics in user-defined projects. See link:https://docs.openshift.com/observability/dedicated/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/observability/rosa/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS).
```

Comment:
Maybe it's something I'm doing wrong, but these two links are not working for me.

Reply:
Oh, I see what I did here; the "observability" should be after Rosa/dedicated. Fixing.